definitions/reports/ldap_auth_source.rb

```diff
@@ -0,0 +1,75 @@
+module Checks
+  module Report
+    class LDAPAuthSource < ForemanMaintain::Report
+      metadata do
+        description 'Checks the use of LDAP auth sources'
+      end
+
+      # Do you use FreeIPA LDAP auth source?
+      # Do you use AD LDAP auth source?
+      # Do you use POSIX LDAP auth source?
+      # Do you use netgroups schema?
+      # Do you disable automatic account creation on any LDAP auth source?
+      # Do you disable user group syncrhonization on any LDAP auth source?
+      # Do you have external user groups mapping?
+      def run
+        self.data = {}
+        data["external_user_group_mapping_count"] = sql_count('external_usergroups')
+        %w[free_ipa posix active_directory].reduce({}) do |_acc, flavor|
+          record_flavor_usage(flavor)
+        end
+      end
+
+      private
+
+      # rubocop:disable Metrics/AbcSize
+      def record_flavor_usage(flavor)
+        query_base = query_base(flavor)
+        data["ldap_auth_source_#{flavor}_count"] = sql_count(query_base)
```

Comment:
is the `query_base` inside `sql_count` the variable? the method? ;)

Reply:
you mean it's better to rename the variable to something else so it's not confusing?

Reply:
yes